### ISSUE TITLE
Add alt text option for all images

### ISSUE THREAD

**Issue by stephenhutson** [Simplero/Roadmap#158]
Any image that is on a front-facing website should have alt text attached to it. It's vitally important for making the web accessible for visually impaired users. So what I'd like to be able to do is go into the media file manager and assign the alt text there, so that anywhere the image is used, that alt text goes with it.
